Functions.Templates/ProjectTemplate_v4.x/CSharp/host.json

```diff
@@ -5,7 +5,8 @@
             "samplingSettings": {
                 "isEnabled": true,
                 "excludedTypes": "Request"
-            }
+            },
+            "enableLiveMetricsFilters": true

```

Comment:
@brettsam wanted to confirm that we need this setting only for in-proc app and also the value should be true (i.e. by default we want live metrics to be _filtered_)

Reply:
we want it for all apps as the logs are really coming from the extensions (which all app types use) -- and yes, we want the filters to apply to live metrics

Reply:
Added to other templates.